Zadanie 16.2 Połączenie Webpacka z Reactem, wstęp do aplikacji Todo/src/containers/App.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import style from './App.css';
+import Title from '../components/Title';
+
+class App extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            data: []
+        };
+    }
+    addTodo(val) {
+        const todo = {
+            text: val,
+            id: uuid.v4()
+        }
+        const data = [...this.state.data, todo];
+        this.setState({data});
+    }
+
+    removeTodo(id) {
+        const remainder = this.state.data.filter(todo => todo.id !==id);
+        this.setState({data: remainder});
+    }
+    render() {
+        return (
+            <div className={style.TodoApp}>
+                <p>Test</p>
+                <Title title={'TodoApp'}/>
```

Comment:
paragraf z testem mozesz usunac, natomiast zrobimy sobie update do `<Title/>`: 
- zamiast `title={'TodoApp'}` napisz sobie `title="TodoApp"` - ten `title` wykorzystasz pozniej w `Title.js` jako `{props.title}`
- dodaj tez atrybut, w ktorym bedziesz przekazywal ilosc todo, czyli `count={this.state.data.length}`